### PR TITLE
[Improve][Docs] resource isolation add update node tags link

### PR DESCRIPTION
**Pull request by Cheun99**
### Purpose of this pull request

update resource isolation add update node tags link https://github.com/apache/seatunnel/pull/7542

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/zh/seatunnel-engine/resource-isolation.md [hunk 1]
```diff
@@ -82,3 +82,7 @@ sink {
 
 ![img.png](../../images/resource-isolation.png)
 
+3. 更新运行中node的tags
+
+获取具体是使用信息，请参考 [更新运行节点的tags](https://seatunnel.apache.org/zh-CN/docs/seatunnel-engine/rest-api/)
```
Comment (Hisoka-X) [inline docs/zh/seatunnel-engine/resource-isolation.md]:
```suggestion
获取具体的使用信息，请参考 [更新运行节点的tags](https://seatunnel.apache.org/zh-CN/docs/seatunnel-engine/rest-api/)
```

File: docs/en/seatunnel-engine/resource-isolation.md [hunk 2]
```diff
@@ -82,3 +82,7 @@ sink {
 
 ![img.png](../../images/resource-isolation.png)
 
+3. update running node tags by rest api
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/resource-isolation.md]:
```suggestion
3. update running node tags by rest api (optional)
```

File: docs/zh/seatunnel-engine/resource-isolation.md [hunk 3]
```diff
@@ -82,3 +82,7 @@ sink {
 
 ![img.png](../../images/resource-isolation.png)
 
+3. 更新运行中node的tags
```
Comment (Hisoka-X) [inline docs/zh/seatunnel-engine/resource-isolation.md]:
```suggestion
3. 更新运行中node的tags （可选）
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @Cheun99 !

**Review by liunaijie — APPROVED** [review]

**Pull request merged**